proxymode/resolver.go

```diff
@@ -16,25 +17,57 @@ type proxyResolver struct {
 	destinationURL url.URL
 }
 
+var acceptedHeaders = map[string]string{
+	"Content-Length": "",
+	"Content-Type":   "",
+	"Last-Modified":  "",
+}
+
+func (pr proxyResolver) pipeRequestProxy(w http.ResponseWriter, remotePath string) error {
+	remoteURL := pr.destinationURL.ResolveReference(&url.URL{Path: remotePath})
+
+	resp, err := http.Get(remoteURL.String())
+	if err != nil {
+		return err
+	}
+	defer resp.Body.Close()
+
+	for header, values := range resp.Header {
+		if len(w.Header().Values(header)) > 0 {
+			continue
+		}
+		if _, ok := acceptedHeaders[header]; !ok {
+			continue
+		}
+		for _, value := range values {
+			w.Header().Add(header, value)
+		}
+	}
```

Comment:
In this loop, just the accepted headers (defined in `acceptedHeaders` map) are added to the response headers.

Here it can be checked all the headers that are received from package-storage.elastic.co:

```
Filtered header X-Goog-Generation:[1658857006542520]
Filtered header X-Goog-Storage-Class:[MULTI_REGIONAL]
Filtered header Access-Control-Allow-Origin:[*]
Filtered header X-Goog-Metageneration:[1]
Filtered header X-Goog-Stored-Content-Encoding:[identity]
Filtered header Age:[360]
Filtered header Date:[Thu, 24 Nov 2022 14:53:42 GMT]
Adding header Content-Type:application/zip
Filtered header (already exists) Cache-Control:[public,max-age=3600]
Filtered header Alt-Svc:[h3=":443"; ma=2592000,h3-29=":443"; ma=2592000]
Filtered header X-Guploader-Uploadid:[ADPycdsyIjbm....]
Filtered header X-Goog-Stored-Content-Length:[655945]
Filtered header X-Goog-Hash:[crc32c=Zx4KuQ== md5=Iqb3+RJove6sG6lrKgaPMw==]
Filtered header Accept-Ranges:[bytes]
Adding header Content-Length:655945
Filtered header Server:[UploadServer]
Adding header Last-Modified:Tue, 26 Jul 2022 17:36:46 GMT
Filtered header Etag:["22a6f7f91268bdeeac1ba96b2a068f33"]
```

Reply:
I will add `date` for completeness.